### DISCUSSION TITLE
gx device not seeing batteries past firmware 1.020230531

### DISCUSSION THREAD

**Discussion by Maxmat99** [Louisvdw/dbus-serialbattery#1067] [Q&A]
hi i did have a search around pinned discussions and couldnt find anything similar....
if i install a firmware version higher than the above the dbs doesnt pick any more than 2 batteries up... any ideas ?
daly bms 6xm12s 3.7
pi 3b 
kwindred batt agregator

**Comment by mr-manuel** [comment]
You already asked in https://github.com/Louisvdw/dbus-serialbattery/discussions/55#discussioncomment-9563079, where I made you note to open a support issue, if you need help. Without detailed data about your system or a glas ball it's hard to know the details. Like you want a detailed answer to solve your problem, we need a detailed description ;-)

Please read carefully the instructions and click on every link when opening the issue. Else you will miss important informations.

**Reply by Maxmat99** [reply]
ah yes sorry i was looking on my phone and flitting between that and computer , and didnt have the link on email!
